build.sh

```diff
@@ -90,21 +97,42 @@ VERSION=$(head -n 1 VERSION)
 case $VERSION in
   *SNAPSHOT)
     echo "Snapshot version (${VERSION}) specified in $(find . -name VERSION): building nightly plugin registry."
-    docker build \
-        -t "${IMAGE}" \
-        -f ${DOCKERFILE} \
-        --build-arg LATEST_ONLY="${LATEST_ONLY}" \
-        --build-arg "USE_DIGESTS=${USE_DIGESTS}" \
-        --target "${TARGET}" .
+    if [ $USE_BUILDAH = true ]
```

Comment:
In some other repo we try to detect with `which` command if docker or podman commands are available so no need to use switch

Reply:
what if both are installed, but the user wants to use a specific one?

Reply:
if it's building I would say users may not care but you could keep flag if you think it's important

I think that it's nice if it's building trying available tools without knowing special flags

Reply:
I am thinking more of the case where someone's machine has docker pre-installed but without root access, in which case they'd want to force the usage of buildah

Reply:
if there's a reason to explicitly ask for podman/buildah when docker is also present, then this approach makes sense.

In other places, we have stuff like:

```
PODMAN=$(command -v podman || true)
if [[ ! -x $PODMAN ]]; then
  echo "[WARNING] podman is not installed."
 PODMAN=$(command -v docker || true)
  if [[ ! -x $PODMAN ]]; then
    echo "[ERROR] docker is not installed. Aborting."; exit 1
  fi
fi
```

https://github.com/redhat-developer/codeready-workspaces-deprecated/blob/crw-2.6-rhel-8/build.sh#L46-L53

Reply:
I forgot about `podman`, I'll adjust the script